config/src/main/java/org/springframework/security/config/annotation/web/AbstractRequestMatcherRegistry.java

```diff
@@ -509,7 +509,8 @@ static class DispatcherServletRequestMatcher implements RequestMatcher {
 		public boolean matches(HttpServletRequest request) {
 			String name = request.getHttpServletMapping().getServletName();
 			ServletRegistration registration = this.servletContext.getServletRegistration(name);
-			Assert.notNull(registration, computeErrorMessage(this.servletContext.getServletRegistrations().values()));
+			Assert.notNull(registration,
```

Comment:
It looks like this might have wandered in from a merge. Can you rebase on `6.3.x` to remove this change from your commit?